providers/apache/hive/src/airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -35,10 +35,17 @@
 from airflow.configuration import conf
 from airflow.exceptions import AirflowException
 from airflow.hooks.base import BaseHook
+from airflow.providers.common.compat.version_compat import AIRFLOW_V_3_0_PLUS
 from airflow.providers.common.sql.hooks.sql import DbApiHook
 from airflow.security import utils
 from airflow.utils.helpers import as_flattened_list
-from airflow.utils.operator_helpers import AIRFLOW_VAR_NAME_FORMAT_MAPPING
+
+if AIRFLOW_V_3_0_PLUS:
+    from airflow.sdk.execution_time.context import AIRFLOW_VAR_NAME_FORMAT_MAPPING
+else:
+    from airflow.utils.operator_helpers import (  # type: ignore[no-redef, attr-defined]
+        AIRFLOW_VAR_NAME_FORMAT_MAPPING,
+    )
```

Comment:
```suggestion
    from airflow.utils.operator_helpers import AIRFLOW_VAR_NAME_FORMAT_MAPPING  # type: ignore[no-redef]
```

Only no-redef not enough?

Reply:
getting static check failures when only no-redef is used:

```
providers/apache/hive/src/airflow/providers/apache/hive/hooks/hive.py:46: error:
Module "airflow.utils.operator_helpers" has no attribute
"AIRFLOW_VAR_NAME_FORMAT_MAPPING"  [attr-defined]
        from airflow.utils.operator_helpers import (  # type: ignore[no-re...
        ^
providers/apache/hive/src/airflow/providers/apache/hive/hooks/hive.py:46: note: Error code "attr-defined" not covered by "type: ignore" comment
Found 6 errors in 5 files (checked 1178 source files)
Error 1 returned
```

Reply:
Cool